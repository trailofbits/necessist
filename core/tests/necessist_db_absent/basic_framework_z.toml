args = ["--no-sqlite", "--root=examples/basic", "--framework=z"]

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 2

[env.add]
TRYCMD="1"
