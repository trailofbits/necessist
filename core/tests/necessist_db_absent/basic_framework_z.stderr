error: invalid value 'z' for '--framework <FRAMEWORK>'
  [possible values: auto, foundry, hardhat-ts, rust]

For more information, try '--help'.
